packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/ip-ip-firewall-game.controler.js

```diff
@@ -123,6 +136,20 @@ export default /* @ngInject */ function IpGameFirewallCtrl(
   function init() {
     getFirewall();
     getRules();
+
+    // pagination
+    const { page, pageSize } = $location.search();
+    $scope.pageNumber = toInteger(page) || 1;
```

Comment:
`+` operator is enough to convert into integer
```suggestion
    $scope.pageNumber = +page || 1;
```